guidelines/index.html

```diff
@@ -1432,30 +1432,30 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
     </ul>
 
-    <p>Exceptions:</p>
+    <p>例外:</p>
     <ul>
-      <li>The focus indicator is determined by the <a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-1" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">user agent</a> and cannot be adjusted by the author, or</li>
-        <li>The focus indicator and the indicator's background color are not modified by the author.</li>
+      <li>フォーカスインジケータが<a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-1" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">ユーザエージェント</a>によって定められており、かつコンテンツ制作者による調整ができない場合。又は、</li>
+        <li>フォーカスインジケータ及びそのインジケータの背景の色を、コンテンツ制作者が変更していない場合。</li>
     </ul>
     
-    <div class="note" role="note" id="issue-container-generatedID-29"><div role="heading" class="note-title marker" id="h-note-29" aria-level="5"><span>注記 1</span></div><p class="">What is perceived as the user interface component or sub-component (to determine enclosure or size) depends on its visual <a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-3" title="rendering of the content in a form to be perceived by users">presentation</a>. The visual presentation includes the component's visible <a href="#dfn-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-content-1" title="コンテンツの構造、提示、及びインタラクションを定義するコード又はマークアップも含めて、ユーザエージェントによって利用者に伝達される情報及び感覚的な体験。">content</a>, border, and component-specific background. It does not include shadow and glow effects outside the component's content, background, or border.</p></div>
+																																																																																													<div class="note" role="note" id="issue-container-generatedID-29"><div role="heading" class="note-title marker" id="h-note-29" aria-level="5"><span>注記 1</span></div><p class="">ユーザインタフェースコンポーネント又はサブコンポーネントとして認識されるもの (輪郭又は大きさを決定づけるもの) は、その視覚的<a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-3" title="利用者が知覚できる形式でコンテンツをレンダリングすること。">提示</a>に依存する。視覚的提示には、コンポーネントの可視<a href="#dfn-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-content-1" title="コンテンツの構造、提示、及びインタラクションを定義するコード又はマークアップも含めて、ユーザエージェントによって利用者に伝達される情報及び感覚的な体験。">コンテンツ</a>、ボーダー、及びコンポーネント固有の背景が含まれる。コンポーネントのコンテンツ、背景、又はボーダーの外側にある、影及び光の効果は含まれない。</p></div>
 
-    <div class="note" role="note" id="issue-container-generatedID-30"><div role="heading" class="note-title marker" id="h-note-30" aria-level="5"><span>注記 2</span></div><p class="">Examples of sub-components that may receive a focus indicator are menu items in an opened drop-down menu, or focusable cells in a grid.</p></div>
+    <div class="note" role="note" id="issue-container-generatedID-30"><div role="heading" class="note-title marker" id="h-note-30" aria-level="5"><span>注記 2</span></div><p class="">フォーカスインジケータを受け取ることができるサブコンポーネントの例としては、開いたドロップダウンメニューの中のメニュー項目、又はグリッド内のフォーカス可能なセル、などがある。</p></div>
```

Comment:
JIS Z 8301:2019 H.3.2.4いわく、
> “又は”などによって併記した最後の語句に，“など”，“その他”などを付けない。

```suggestion
    <div class="note" role="note" id="issue-container-generatedID-30"><div role="heading" class="note-title marker" id="h-note-30" aria-level="5"><span>注記 2</span></div><p class="">フォーカスインジケータを受け取ることができるサブコンポーネントの例としては、開いたドロップダウンメニューのメニュー項目、グリッド内のフォーカス可能なセルなどがある。</p></div>
```